source/transactions-convenient-api/tests/README.rst

```diff
@@ -28,16 +28,33 @@ Test Format
 
 Each YAML file has the following keys:
 
+- ``runOn`` (optional): An array of server version and/or topology requirements
+  for which the tests can be run. If the test environment satisfies one or more
+  of these requirements, the tests may be executed; otherwise, this file should
+  be skipped. If this field is omitted, the tests can be assumed to have no
+  particular requirements and should be executed. Each element will have some or
+  all of the following fields:
+
+  - ``minServerVersion`` (optional): The minimum server version (inclusive)
+    required to successfully run the tests. If this field is omitted, it should
+    be assumed that there is no lower bound on the required server version.
+
+  - ``maxServerVersion`` (optional): The maximum server version (exclusive)
+    against which the tests can be run successfully. If this field is omitted,
+    it should be assumed that there is no upper bound on the required server
+    version.
+
+  - ``topology`` (optional): An array of server topologies against which the
```

Comment:
I think `topology` should be plural, since its value is an array.

Reply:
Let's make a SPEC ticket to do this work later since `topology` is currently used in a number of other test formats. I agree with you it should be plural, but I think this should all happen at the same time.

Reply:
https://jira.mongodb.org/browse/SPEC-1254